src/Expect.elm

```diff
@@ -0,0 +1,367 @@
+module Expect exposing (Expectation, pass, fail, getFailure, equal, notEqual, atMost, lessThan, greaterThan, atLeast, true, false, onFail)
+
+{-| Determining whether tests pass or fail.
+
+## Quick Reference
+
+* [`equal`](#equal) `(arg2 == arg1)`
+* [`notEqual`](#notEqual) `(arg2 /= arg1)`
+* [`lessThan`](#lessThan) `(arg2 < arg1)`
+* [`atMost`](#atMost) `(arg2 <= arg1)`
+* [`greaterThan`](#greaterThan) `(arg2 > arg1)`
+* [`atLeast`](#atLeast) `(arg2 >= arg1)`
+* [`true`](#true) `(arg == True)`
+* [`false`](#false) `(arg == False)`
+
+## Basic Expectations
+
+@docs Expectation, equal, notEqual
+
+## Comparisons
+
+@docs lessThan, atMost, greaterThan, atLeast
+
+## Booleans
+
+@docs true, false
+
+## Customizing
+
+@docs pass, fail, onFail, getFailure
+-}
+
+import Test.Expectation
+import String
+
+
+{-| The result of a single test run: either a [`pass`](#pass) or a
+[`fail`](#fail).
+-}
+type alias Expectation =
+    Test.Expectation.Expectation
+
+
+{-| Passes if the arguments are equal.
+
+    Expect.equal 0 (List.length [])
+
+    -- Passes because (0 == 0) is True
+
+Failures resemble code written in pipeline style, so you can tell
+which argument is which:
+
+    -- Fails because (0 == 5) is False
+    List.length []
+        |> Expect.equal 5
+
+
+    {-
+
+    0
+    ╷
+    │ Expect.equal
+    ╵
+    5
```

Comment:
Maybe use an example with larger bad values? Like two strings that are not equal. This way, it is clearer why the layout is like this. As is, it's like "why not just put them side-by-side?"